native/c/zds.hpp

```diff
@@ -24,11 +24,15 @@ struct ZDSMem
   // std::string dsorg;
 };
 
+#define MIGRATE_YES "YES"
+#define MIGRATE_NO "NO"
+
 struct ZDSEntry
 {
   std::string name;
   std::string dsorg;
   std::string volser;
+  std::string migr;
```

Comment:
This makes the DS entry consistent with FTP ([zos-node-accessor](https://github.com/IBM/zos-node-accessor/blob/0431d0d244aee348d71ec7357f2fd17c1b58fc69/src/interfaces/DatasetEntry.ts#L41)) 🥳 

If it's all the same to y'all... could we make this a boolean value? 
Unless we believe that we may want to use it for something other than a `yes|no` value. 😋

Reply:
I agree, I'll change this

Reply:
fixt